### DISCUSSION TITLE
self-hosted in docker dashboard alternative

### DISCUSSION THREAD

**Discussion by X4V1** [supabase/supabase#1299] [Questions] [answered]
I'm discovering Supabase and I'm trying to use the self-hosted version in docker.
I ejected the supabase docker compose file and started it with all the config left at default.
It seems to start correctly (can connect to supabase with the js client).
But there is something I don't understand, everywhere in the documentation there are reference to the dashboard (which does not seems to included in the self-hosted version).
So I would like to know how we can configure those settings without the graphical interface. 
Is there any alternative to the graphical interface (the documentation is not mentioning anything else).

For instance I would like to turn off the possibility to signup and only allow invitation.
I don't find anything in the doc explaining how to turn off signup at runtime without using the dashboard, it is possible?
And when I try to send an Invitation I'm getting:
``{
  "message": "User not allowed",
  "status": 401
}``
I'm using the following code:
```
  const supabase = createClient('supabaseUrl', 'anon-public-key');
  await supabase.auth.api.signUpWithEmail('example@mail.com', 'password');
  await supabase.auth.api.signInWithEmail('example@mail.com', 'password');
  await supabase.auth.api.inviteUserByEmail('another@mail.com');
```
Am I missing something ? I tried to access the supabase url to see if there is a dashboard available for the self hosted version but I got a message saying 
``
{
message: "no Route matched with those values"
}``

Am I missing something ?

Also, is it possible to control a self-hosted database with the online dashboard ?

Kind regards,

Xavier

**Comment by soedirgo** [comment, marked as answer]
Hi @X4V1, the dashboard isn't open source yet (we're working on this!), more info [here](https://supabase.io/docs/guides/self-hosting#before-you-begin).

For now, to disable signups, you need to set the [`GOTRUE_DISABLE_SIGNUP`](https://supabase.io/docs/gotrue/server/about#configuration) environment variable in the Compose file to `'true'`. If you've created the `supabase-auth` container already, you might need to pass `--force-recreate` to docker-compose.

I'll look into the invitation error, possibly related to https://github.com/supabase/cli/issues/22.

**Reply by X4V1** [reply]
Thank you for your feedback. I will wait for the dashboard before diving deeper with supabase then :)

**Reply by awalias** [reply]
hey @X4V1 , @kiwicopple is building this in public if you wanted to follow along or try the dev versions: https://www.youtube.com/watch?v=LHYrqBb4q9I
